pages/components/navigation/navigation.html

```diff
@@ -72,13 +72,21 @@ <h1 class="showcase-header">Simple Navigation</h1>
 						<div class="badge-icon">9</div>
 					</div>
 
-					<img
-						class="avatar avatar-sm"
-						src="https://media-exp1.licdn.com/dms/image/C4E03AQEm0ZBjaIr1hg/profile-displayphoto-shrink_200_200/0/1610261504692?e=1649894400&v=beta&t=kKO77EwqflEDsrZf5eG7xC7ZUB4hD_BkZuRJyFiFHbI"
-					/>
+					<div class="profile-modal-holder">
+						<img
+							class="avatar avatar-sm"
+							src="https://media-exp1.licdn.com/dms/image/C4E03AQEm0ZBjaIr1hg/profile-displayphoto-shrink_200_200/0/1610261504692?e=1649894400&v=beta&t=kKO77EwqflEDsrZf5eG7xC7ZUB4hD_BkZuRJyFiFHbI"
```

Comment:
can't you import image from images folder (if you have any) ?

Reply:
The images is not in my folders that's why.